packages/ui/src/i18n/dictionaries/authenticator/defaultTexts.ts

```diff
@@ -12,74 +12,74 @@ export const defaultTexts = {
-  ENTER_EMAIL: 'Enter your Email',
-  ENTER_FAMILY_NAME: 'Enter your Family Name',
-  ENTER_GIVEN_NAME: 'Enter your Given Name',
-  ENTER_MIDDLE_NAME: 'Enter your Middle Name',
-  ENTER_NAME: 'Enter your Name',
-  ENTER_NICK_NAME: 'Enter your Nickname',
-  ENTER_PASSWORD: 'Enter your Password',
-  ENTER_PHONE_NUMBER: 'Enter your Phone Number',
-  ENTER_PREFERRED_USERNAME: 'Enter your Preferred Username',
+  ENTER_CONFIRMATION_CODE: 'Enter your confirmation code',
+  ENTER_EMAIL: 'Enter your email',
+  ENTER_FAMILY_NAME: 'Enter your family name',
+  ENTER_GIVEN_NAME: 'Enter your given name',
+  ENTER_MIDDLE_NAME: 'Enter your middle name',
+  ENTER_NAME: 'Enter your name',
+  ENTER_NICK_NAME: 'Enter your nickname',
+  ENTER_PASSWORD: 'Enter your password',
+  ENTER_PHONE_NUMBER: 'Enter your phone number',
+  ENTER_PREFERRED_USERNAME: 'Enter your preferred username',
   ENTER_USERNAME: 'Enter your username',
-  FAMILY_NAME: 'Family Name',
-  GIVEN_NAME: 'Given Name',
-  FORGOT_PASSWORD: 'Forgot Password?',
+  FAMILY_NAME: 'Family name',
+  GIVEN_NAME: 'Given name',
+  FORGOT_PASSWORD: 'Forgot password?',
   FORGOT_YOUR_PASSWORD: 'Forgot your password?',
   HIDE_PASSWORD: 'Hide password',
   LOADING: 'Loading',
   LOGIN_NAME: 'Username',
-  MIDDLE_NAME: 'Middle Name',
+  MIDDLE_NAME: 'Middle name',
   NAME: 'Name',
   NICKNAME: 'Nickname',
   NEW_PASSWORD: 'New password',
   OR: 'or',
   PASSWORD: 'Password',
-  PHONE_NUMBER: 'Phone Number',
-  PREFERRED_USERNAME: 'Preferred Username',
+  PHONE_NUMBER: 'Phone number',
+  PREFERRED_USERNAME: 'Preferred username',
   PROFILE: 'Profile',
-  RESEND_CODE: 'Resend Code',
+  RESEND_CODE: 'Resend code',
   RESET_PASSWORD_HEADING: 'Reset your password',
-  RESET_PASSWORD: 'Reset Password',
+  RESET_PASSWORD: 'Reset password',
   SEND_CODE: 'Send code',
   SENDING: 'Sending',
   SETUP_TOTP: 'Setup TOTP',
   SHOW_PASSWORD: 'Show password',
   SIGN_IN_BUTTON: 'Sign in',
-  SIGN_IN_TAB: 'Sign In',
-  SIGN_IN_WITH_AMAZON: 'Sign In with Amazon',
-  SIGN_IN_WITH_APPLE: 'Sign In with Apple',
-  SIGN_IN_WITH_FACEBOOK: 'Sign In with Facebook',
-  SIGN_IN_WITH_GOOGLE: 'Sign In with Google',
+  SIGN_IN_TAB: 'Sign in',
+  SIGN_IN_WITH_AMAZON: 'Sign in with Amazon',
+  SIGN_IN_WITH_APPLE: 'Sign in with Apple',
+  SIGN_IN_WITH_FACEBOOK: 'Sign in with Facebook',
+  SIGN_IN_WITH_GOOGLE: 'Sign in with Google',
   SIGN_IN: 'Sign in to your account',
   SIGN_UP_BUTTON: 'Create a new account',
   SIGNING_IN_BUTTON: 'Signing in',
   SKIP: 'Skip',
   SUBMIT: 'Submit',
   SUBMITTING: 'Submitting',
   UPPERCASE_COPY: 'COPY',
-  VERIFY_CONTACT: 'Verify Contact',
+  VERIFY_CONTACT: 'Verify contact',
   VERIFY_HEADING: 'Account recovery requires verified contact information',
   VERIFY: 'Verify',
-  WE_EMAILED: 'We Emailed You',
-  WE_SENT_CODE: 'We Sent A Code',
-  WE_TEXTED: 'We Texted You',
+  WE_EMAILED: 'We emailed you',
+  WE_SENT_CODE: 'We sent a code',
+  WE_TEXTED: 'We texted you',
```

Comment:
smallest nit in the world and I know it's not in scope, but should `We sent a code` be more like `We sent you a code` to align with the other texts?